test/WebJobs.Script.Tests/ContentEndToEndTests.cs

```diff
@@ -19,9 +19,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Tests.Content
 {
-    public class NodeTextPlain : ContentTests
+    public class NodeTextPlain : ContentTests<NodeContentFixture>
```

Comment:
@fabiocav hm looks like you're right about collection fixtures, interesting.  Since fixture startup time doesn't show up in the test run time it's hard to see whether its per class or per run